### ISSUE TITLE
Rule Change: Treat equivalent keyframe selectors as duplicates

### ISSUE THREAD

**Issue by Pixel998** [eslint/css#508]
### What rule do you want to change?

`no-duplicate-keyframe-selectors`

### What change do you want to make?

Generate more warnings

### How do you think the change should be implemented?

A new default behavior

### Example code

```css
/* eslint css/no-duplicate-keyframe-selectors: "error" */

@keyframes fade-in {
  from {
    opacity: 0;
  }

  0% {
    opacity: 0.25;
  }

  to {
    opacity: 1;
  }

  100% {
    opacity: 0.75;
  }
}
```

### What does the rule currently do for this code?

The rule does not report any errors because it compares `from`, `0%`, `to`, and `100%` as distinct selectors.

### What will the rule do after it's changed?

The rule will report `0%` as a duplicate of `from` and `100%` as a duplicate of `to`, because each pair identifies the same keyframe position.

### Participation

- [x] I am willing to submit a pull request to implement this change.

### AI acknowledgment

- [x] I did not use AI to generate this issue report.
- [ ] (If the above is not checked) I have reviewed the AI-generated content before submitting.

### Additional comments

https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/Selectors/Keyframe_selectors#values

**Comment by DMartens** [comment]
I agree that the special keywords `to` and `from` should be normalized and think we should treat this as a bugfix rather than a feature.
